DOCSP-48487 Move usage examples (#1071)
mongodb/docs-node: source/crud/query/retrieve.txt

        ...
      ]

.. _node-usage-findone:

findOne() Usage Example
~~~~~~~~~~~~~~~~~~~~~~~

.. include:: /includes/crud/example-intro.rst

The following full file example finds a single document from the ``movies``
collection. It uses the following parameters:

> - A **query document** that configures the query to return only
>   movies with the title of exactly the text ``'The Room'``.
> 
> - A **sort** that organizes matched documents in descending order by
>   rating, so if our query matches multiple documents the returned
>   document will be the document with the highest rating.
> 
> - A **projection** that explicitly excludes the ``_id`` field from
>   returned documents and explicitly includes only the ``title`` and
>   ``imdb`` object (and its embedded fields).

Reviewer: S: style guide - https://www.mongodb.com/docs/meta/style-guide/style/lists/#:~:text=Avoid%20using%20articles%20(a%2C%20an%2C%20the)%20to%20start%20list%20items.

Rephrase these list items to omit articles
ie
```suggestion
- **Filter** that matches documents in which the ``title`` value is ``"The Room"``.

- **Sort** that organizes matched documents in descending order by
  rating, so if our query matches multiple documents the returned
  document will be the document with the highest rating.

- **Projection** that explicitly excludes the ``_id`` field from
  returned documents and explicitly includes only the ``title`` and
  ``imdb`` object (and its embedded fields).
```

.. _node-driver-findone-usage-example-code-snippet:

.. tabs::

  .. tab:: JavaScript
     :tabid: javascript

     .. literalinclude:: /code-snippets/usage-examples/findOne.js
        :language: javascript

  .. tab:: TypeScript
     :tabid: typescript

     .. literalinclude:: /code-snippets/usage-examples/findOne.ts
        :language: typescript

Running the preceding example results in the following output:

.. code-block:: javascript
   :copyable: false

   { title: 'The Room', imdb: { rating: 3.5, votes: 25673, id: 368226 } }

.. _node-usage-find:

find() Example: Full File
~~~~~~~~~~~~~~~~~~~~~~~~~

.. include:: /includes/crud/example-intro.rst

The following full file example finds documents from the ``movies`` collection. It
uses the following parameters:

- A **query document** that configures the query to return only
  movies with a runtime of less than 15 minutes.

- A **sort** that organizes returned documents in ascending order by
  title (alphabetical order in which "A" comes before "Z" and "1" before
  "9").

- A **projection** that explicitly excludes the ``_id`` field from
  returned documents and explicitly includes only the ``title`` and

>   ``imdb`` object (and its embedded fields).

Reviewer: S: same feedback as for the findONe example to remove articles from this list

.. _node-driver-find-usage-example-code-snippet:

.. tabs::

  .. tab:: JavaScript
     :tabid: javascript

     .. literalinclude:: /code-snippets/usage-examples/find.js
        :language: javascript
        :linenos:

  .. tab:: TypeScript
     :tabid: typescript

     .. literalinclude:: /code-snippets/usage-examples/find.ts
        :language: typescript
        :linenos:

Running the preceding example, you see the following output:

.. code-block:: javascript
   :copyable: false

   { title: '10 Minutes', imdb: { rating: 7.9, votes: 743, id: 339976 } }
   { title: '3x3', imdb: { rating: 6.9, votes: 206, id: 1654725 } }
   { title: '7:35 in the Morning', imdb: { rating: 7.3, votes: 1555, id: 406501 } }
   { title: '8', imdb: { rating: 7.8, votes: 883, id: 1592502 } }
   ...

> The ``sort`` and ``projection`` options can also be specified as methods
> (``sort()`` and ``project()``) chained to the ``find()`` method.
> The following two commands are equivalent:

Reviewer: ```suggestion
You can also specify ``sort`` and ``projection`` options as methods by
chaining the ``sort()`` and ``projection()`` methods to the ``find()`` method.
The following two commands are equivalent:
```

Additional Information
~~~~~~~~~~~~~~~~~~~~~~

For a runnable example of the ``watch()`` method, see the :ref:`examples

> <node-usage-watch:>` section in the :ref:`node-change-streams` guide.

Reviewer: ```suggestion
<node-usage-watch>` section in the :ref:`node-change-streams` guide.
```
